plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/converter/GlueInputConverter.java

```diff
@@ -64,12 +66,16 @@ public static DatabaseInput convertDatabase(Database database)
 
     public static TableInput convertTable(Table table)
     {
+        TableInput input = new TableInput();
+        Map<String, String> tableParameters = table.getParameters();
         Optional<String> comment = Optional.ofNullable(table.getParameters().get(TABLE_COMMENT));
```

Comment:
move comemnt variable inside the if

Reply:
or, keep `comment.ifPresent(input::setDescription);` line where it was, initialize `Optional<String> comment = Optional.empty();` and fill it only when not a view and not a materialized view